### DISCUSSION TITLE
Problem with saving a model using mlflow

### DISCUSSION THREAD

**Discussion by JulianPerez852** [explosion/spaCy#12296] [Help: Coding & Implementations]
Hi everyone!!
I have an issue when I tried to save a mlflow model, I am using the same code that the documentation and my model will be storage in azure ML, but when the code run, that happens.

The code is:
`import mlflow
import mlflow.spacy

nlp = spacy.load('models/nlp')
with mlflow.start_run(run_name='Spacy'):
    mlflow.set_tag('model_flavor', 'spacy')
    mlflow.spacy.log_model(spacy_model=nlp, artifact_path='model_spacy')
    mlflow.log_metric(('accuracy', 0.72))
    my_run_id = mlflow.active_run().info.run_id

model_uri = f'runs:/{my_run_id}/model_spacy'`

The error:
![image](https://user-images.githubusercontent.com/50587788/219661371-accb4e86-1131-4a1d-b857-4edd6de6a8d3.png)

It means "The process don't have access to the file, because is using by another process"

Thanks for your help!

**Comment by rmitsch** [comment]
Hi @JulianPerez852, which versions of `spacy` and `mlflow` are you using?

**Reply by JulianPerez852** [reply]
Hi!!

spacy==3.5.0

mlflow==2.0.1

The model was training with spacy 3.4 but it works when I load It.

**Comment by svlandeg** [comment]
Hi! The code you shared are all calls into `mlflow`'s functionality, which is where I assume you got the code snippet from? So it's probably better to ask on their tracker?
